comments.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * The template for displaying comments
+ *
+ * The area of the page that contains both current comments
+ * and the comment form.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Sixteen
+ * @since Twenty Sixteen 1.0
```

Comment:
I'm thinking we should update these header comments to reflect the ALPS theme info. Keeping it as Twenty Sixteen might get confusing.